docs/evaluate/temporal-cloud/sla.mdx

```diff
@@ -54,7 +54,7 @@ Our internal alerting system is based on a [service level objective](https://en.
 When we receive an alert that an SLO is not being met, we page our on-call engineers, which often means that issues are resolved before they become noticeable.
 
 Internally, our components are distributed across a minimum of three availability zones per region.
-We implement a cell architecture.
+We implement a [cell architecture](https://docs.aws.amazon.com/wellarchitected/latest/reducing-scope-of-impact-with-cell-based-architecture/what-is-a-cell-based-architecture.html).
```

Comment:
We've blogged about Cells https://temporal.io/blog/two-years-in#scale,  https://temporal.io/blog/building-durable-cloud-control-systems-with-temporal#implementing-the-data-plane-a-cell-based-architecture  - we should have a first-class definition of what a Temporal Cloud cell is in our docs. Not to expand scope here to adding a full Cloud architecture page (although I do think we should have one eventually) - maybe add a section on /cloud/service-availability and link to that?